src/AcmTable/AcmTable.tsx

```diff
@@ -172,16 +173,21 @@ export interface AcmTableProps<T> {
     setSearch?: (search: string) => void
     sort?: ISortBy | undefined
     setSort?: (sort: ISortBy | undefined) => void
+    showToolbar?: boolean
```

Comment:
Richie wanted a way to hide this when using the table for other things, so I'm just throwing it in with these changes.

Reply:
> Row actions are duplicated on the expanded content:
> ![image](https://user-images.githubusercontent.com/42188127/113873487-dd57e600-9782-11eb-8ca6-a4709a3dc633.png)

I know cluster doesn't use row actions, so this may not be relevant. App will need to switch the a similar mechanism with the actions in the rows themselves, so that they can be removed for the group header row and customized for Argo vs. ACM apps.

Reply:
Not sure why this comment went here??

Reply:
Yeah this will happen if you use the `rowActions` props; because the PF is treating all the rows the same.  For cluster UI, because of the RBAC on the menu and other things, we had to not use the `rowActions` prop and instead create a column with empty name that houses our own custom dropdown